prov/psm/configure.m4

```diff
@@ -46,7 +46,6 @@ AC_DEFUN([FI_PSM_CONFIGURE],[
 	psm_CPPFLAGS="$CPPFLAGS $psm_CPPFLAGS"
 	psm_LDFLAGS="$LDFLAGS $psm_LDFLAGS"
 	psm_LIBS="$LIBS $psm_LIBS"
```

Comment:
It would be tricky to separate the PSM-specifc part from the main set. We could refactor the common configure macros later to make this easier.